app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -48,13 +53,38 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+if [ -z "$secondaryArchSuffix" ]; then
+SUMMARY_dictionaries="Built-in dictionaries for WordGrinder"
+ARCHITECTURES_dictionaries="any"
+PROVIDES_dictionaries="
+	wordgrinder_dictionaries = $portVersion
+	"
+REQUIRES_dictionaries="
+	wordgrinder == $portVersion base
```

Comment:
After all, there's no need to make wordgrinder_dictionaries depend on wordgrinder. We can also completely remove the `SUPPLEMENTS_dictionaries`.
```diff
 REQUIRES_dictionaries="
-	wordgrinder == $portVersion base
 	"
-SUPPLEMENTS_dictionaries="
-	wordgrinder
-	"
```

Also, could you please add a blank line before `BUILD_REQUIRES` ?
Thanls!